Taskfile.yml

```diff
@@ -182,20 +184,24 @@ tasks:
     cmds:
       - "rm -rf '{{.OUTPUT_DIR}}'"
       - "mkdir -p '{{.OUTPUT_DIR}}'"
-      - "meteor build --directory '{{.OUTPUT_DIR}}'"
+      - "PATH='{{.G_METEOR_BUILD_DIR}}':$PATH meteor build --directory '{{.OUTPUT_DIR}}'"
```

Comment:
Since `meteor` is used in this task, let's add it to `deps`. To ensure meteor doesn't get downloaded multiple times, let's set the `run: "once"` property on the `meteor` task.